assets/js/admin/orders/order-overview/models/state.js

```diff
@@ -154,11 +122,14 @@ export const State = Backbone.Model.extend(
 				return this.get( 'order' ).tax;
 			}
 
+			const { number } = this.get( 'formatters' );
 			const items = this.get( 'items' ).models;
+			const adjustments = this.get( 'adjustments' ).getByType( 'fee' );
 
-			return items.reduce(
+			return [ ...items, ...adjustments ].reduce(
 				( amount, item ) => {
-					return amount += +item.get( 'tax' );
+					// Perform early rounding of tax amounts to match storefront calculations.
+					return amount += +number.format( item.get( 'tax' ) );
```

Comment:
Actually, I'm not 100% sure what to do here. I think the rounding does need to be reverted, so that the order data is saved consistently with the storefront, although the final amounts end up being different. However, with my test order discrepancies, this results in the refund process breaking, because the maximum refundable amount is $63.91, as the JS calculated the taxes, but the way the refund amounts add up (by item), it wants to refund $63.92 and there is a validation error.